includes/rest/class-inbox.php

```diff
@@ -108,7 +108,7 @@ public static function user_inbox_get( $request ) {
 		$json->id = \home_url( \add_query_arg( null, null ) );
 		$json->generator = 'http://wordpress.org/?v=' . \get_bloginfo_rss( 'version' );
 		$json->type = 'OrderedCollectionPage';
-		$json->partOf = \get_rest_url( null, "/activitypub/1.0/users/$user_id/inbox" ); // phpcs:ignore
+		$json->partOf = get_rest_url_by_path( sprintf( 'users/%d/inbox', $user_id ) ); // phpcs:ignore
```

Comment:
`get_rest_url_by_path` is in the `Activitypub` namespace, so you have to add `use function Activitypub\get_rest_url_by_path` as the top of the class.